tests/test_module.py

```diff
@@ -462,6 +462,35 @@ def test_range_with_agg_query(self):
                 assert r.execute_command('TS.range', 'tester', start_ts, start_ts + samples_count, 'AGGREGATION',
                                               'count', -1)
 
+    def test_range_with_agg_query_max(self):
+        start_ts = 4L
+        samples_count = 5
+        with self.redis() as r:
+            assert r.execute_command('TS.CREATE', 'tester')
+            self._insert_data(r, 'tester', start_ts, samples_count, 1)
+            expected_result = [[5L, '4']]
+            actual_result = r.execute_command('TS.range', 'tester', 5, 8, 'AGGREGATION',
+                                              'SUM', 4)
+            assert expected_result == actual_result
+
+            actual_result = r.execute_command('TS.range', 'tester', 5, 8, 'AGGREGATION',
+                                              'SUM', 10)
+            assert expected_result == actual_result
+
+            with pytest.raises(redis.ResponseError) as excinfo:
+                assert r.execute_command('TS.range', 'tester', 5, 8, 'AGGREGATION',
+                                              'SUM', -1)
+            
+    def test_range_start_ts_based(self):
```

Comment:
@filipecosta90 
This test shows the issue with the suggested correction.
Let's discuss this in the next weekly.